disnake/channel.py

```diff
@@ -2423,6 +2424,7 @@ async def create_thread(
         embeds: List[Embed] = MISSING,
         file: File = MISSING,
         files: List[File] = MISSING,
+        suppress_embeds: bool = MISSING,
```

Comment:
parameter is missing in documentation

Reply:
https://github.com/DisnakeDev/disnake/pull/474/commits/d6c2b836e3514a602eaad88369d072cc0eeaac9c